packages/manager/modules/iam/src/components/actionSelect/ActionTrees.class.js

```diff
@@ -95,14 +100,62 @@ export default class ActionTrees extends Array {
     return { action: newAction, created: true };
   }
 
+  /**
+   * Check all actions matching to a regex action
+   * If some actions exist, they're tagged as embedded
+   * @param {string} action
+   * @param {Boolean} selected
```

Comment:
Comments says that the function takes a selected param, but it doesn't seem so. Maybe remove it?

Reply:
good catch! Fixed